runtime_test.go

```diff
@@ -1,27 +1,27 @@
-package service
```

Comment:
If we don't need this.. lets remove this for now.

Reply:
this service_test.go The file doesnt exist anymore

Reply:
i've deleted it already. atleast locally.